.vscode/settings.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tailwindCSS.classAttributes": [
+    "class",
+    "className",
+    "ngClass",
+    "[\\S]*[cC]lass[\\S]*"
+  ],
```

Comment:
These are settings for react and angular right? We don't need them I guess?

Reply:
`className` is used in React but is not restricted to it

Have added these patterns to allow tailwind to give details of class on hover when the name of the property follows either of the regex and not `class` only which is the default behavior